beetsplug/deezer.py

```diff
@@ -75,13 +75,12 @@ def album_for_id(self, album_id):
         deezer_id = self._get_id("album", album_id, self.id_regex)
         if deezer_id is None:
             return None
-
-        album_data = requests.get(self.album_url + deezer_id).json()
```

Comment:
Please correct me if I’m wrong, but it seems like we lost this layer of error checking. That is, it seems like we should still check for the “error” key in the JSON response?

Reply:
The try-except block should capture it. If there is an `error` key, it should be caught in `except` block. It is possible that we may have to capture other exceptions (once we know which ones), but that is easy.

Reply:
Hmm; I don't think I quite see the idea here… what exception will be raised when there is an `error` key? (Is there some other code that checks for the existence of this key, since it doesn't seem to happen here?)

Reply:
Ok...I see now what I was missing. I updated the code to handle the `error` key.